patronload/config.py

```diff
@@ -31,3 +35,31 @@ def configure_sentry() -> str:
         sentry_sdk.init(sentry_dsn, environment=env)
         return f"Sentry DSN found, exceptions will be sent to Sentry with env={env}"
     return "No Sentry DSN found, exceptions will not be sent to Sentry"
+
+
+def load_config_values() -> dict:
```

Comment:
It looks like the default config values never get used, since get_required_env_variable raises an error if it can't find the value.

Reply:
You can use something like the `test_env` fixture in `conftest.py` in my latest PR,  I thought I didn't need after adding the default values but it turns out I did

Reply:
Do we actually need the check for required env variables in this app? I'm not sure we do. It was present in alma-scripts because 1) the app and infrastructure were complicated, and 2) the sap invoice process has some very strict configuration requirements. The patronload process is more straightforward and simply won't work correctly if the required variables aren't present.

Reply:
I would prefer to have a reminder at the start of the process to set anything that is missing since this is a simple function. You're correct about `LOG_LEVEL` being optional so I'll shift that.

Reply:
Sure, I'd still argue that adding an extra function for that is unneeded complexity. In alma-scripts we were configuring from both ENV and SSM which meant we had to build in extra checks. Now that we're getting everything from env, why not just use `os.environ[key]` and let the default `KeyError` bubble up?

Reply:
Pushed a new commit to address this